spec/paths/authentication-tokens@{token}@exchange.yaml

```
@@ -9,6 +9,7 @@ post:
   tags:
     - Customer Authentication
   summary: Exchange
+  operationId: PostAuthenticationTokenExchangeRequest
```

```suggestion
  operationId: PostAuthenticationTokenExchange
```
`Request` is not necesary Request